terraform/examples/rds/main.tf

```diff
@@ -0,0 +1,29 @@
+module "rds_instance" {
+  source                       = "../../../modules/rds"
+  
+  aws_region                   = var.aws_region
```

Comment:
no need to specify aws region in db , regions are defined in aws provider

Reply:
resolved